app/controllers/redirect/redirect_controller.rb

```diff
@@ -8,8 +8,13 @@ def location_params
       params.permit(*PERMITTED_LOCATION_PARAMS).to_h.symbolize_keys
     end
 
-    def redirect_to_and_log(url, event: Analytics::EXTERNAL_REDIRECT)
-      analytics.track_event(event, redirect_url: url, **location_params)
+    def redirect_to_and_log(url, event: nil, tracker_method: analytics.method(:external_redirect))
+      if event
+        # Once all events have been moved to tracker methods, we can remove the event: param
```

Comment:
if Alex's PR merges first, I think I can do that in this PR, https://github.com/18F/identity-idp/pull/6159/files#diff-d09b915480bf8b8ebafd39296979f8f3ab0c52293f029efd8ba1369536b3c855R13